pkg/corerp/datamodel/container.go

```diff
@@ -116,16 +116,33 @@ type ConnectionProperties struct {
 
 // Container - Definition of a container.
 type Container struct {
-	Image           string                      `json:"image,omitempty"`
-	ImagePullPolicy string                      `json:"imagePullPolicy,omitempty"`
-	Env             map[string]string           `json:"env,omitempty"`
-	LivenessProbe   HealthProbeProperties       `json:"livenessProbe,omitempty"`
-	Ports           map[string]ContainerPort    `json:"ports,omitempty"`
-	ReadinessProbe  HealthProbeProperties       `json:"readinessProbe,omitempty"`
-	Volumes         map[string]VolumeProperties `json:"volumes,omitempty"`
-	Command         []string                    `json:"command,omitempty"`
-	Args            []string                    `json:"args,omitempty"`
-	WorkingDir      string                      `json:"workingDir,omitempty"`
+	Image           string                         `json:"image,omitempty"`
+	ImagePullPolicy string                         `json:"imagePullPolicy,omitempty"`
+	Env             map[string]EnvironmentVariable `json:"env,omitempty"`
+	LivenessProbe   HealthProbeProperties          `json:"livenessProbe,omitempty"`
+	Ports           map[string]ContainerPort       `json:"ports,omitempty"`
+	ReadinessProbe  HealthProbeProperties          `json:"readinessProbe,omitempty"`
+	Volumes         map[string]VolumeProperties    `json:"volumes,omitempty"`
+	Command         []string                       `json:"command,omitempty"`
+	Args            []string                       `json:"args,omitempty"`
+	WorkingDir      string                         `json:"workingDir,omitempty"`
+}
+
+// EnvironmentVariable - Environment variable for the container
+type EnvironmentVariable struct {
+	Value     *string                       `json:"value,omitempty"`
+	ValueFrom *EnvironmentVariableReference `json:"valueFrom,omitempty"`
```

Comment:
Can we also add documentation for the properties? Maybe with some examples?

Reply:
Do you mean in the code or in the documentation?

Reply:
kind of like:

```suggestion
	// Value is the property for ... Example is ...
	Value     *string                       `json:"value,omitempty"`
	// ValueFrom is the property for ... Example is ...
	ValueFrom *EnvironmentVariableReference `json:"valueFrom,omitempty"`
```